database/mssql/scripts/dbscripts/PSP_PIMS_S16_00/Alter Up/01_PSP_PIMS_S15_01_Alter_Up.sql

```diff
@@ -1311,16 +1332,6 @@ GO
 -- Rebuild table dbo.PIMS_LEASE_HIST
 PRINT N'Rebuild table dbo.PIMS_LEASE_HIST'
 GO
-ALTER TABLE [dbo].[PIMS_LEASE_HIST]
```

Comment:
Why drop these constraints when we are completing replacing the table below? Also, once again Aqua is trying to delete constraints based on generated ids that won't be consistent across database instances.